### DISCUSSION TITLE
GermaNet for spaCy?

### DISCUSSION THREAD

**Discussion by Ablesius** [explosion/spaCy#6341] [New Features & Project Ideas]
## Feature description

There is [spacy-wordnet](https://spacy.io/universe/project/spacy-wordnet), and there's [pygermanet](https://pypi.org/project/pygermanet/). I'm not sure whether it's possible already to access GermaNet features via spaCy directly, but it would surely be nice (for those of us who are lucky enough to have access to GermaNet).

## Could the feature be a [custom component](https://spacy.io/usage/processing-pipelines#custom-components) or [spaCy plugin](https://spacy.io/universe)?
If so, we will tag it as [`project idea`](https://github.com/explosion/spaCy/labels/project%20idea) so other users can take it on.

To answer that question: I think both could be possible.

**Comment by adrianeboyd** [comment]
Sorry, we don't have a license to use GermaNet in spacy and this would be a better fit for a custom extension.

If you want to access the GermaNet data through `pygermanet` or another module, you could write a  component that adds ways to access the data through using custom extensions, for example as custom token extensions? If someone developed this, we would be happy to add it to spacy universe like `spacy-wordnet`: https://spacy.io/universe/

**Comment by Ablesius** [comment]
yes, that's fine, thank you for the pointer :) I'll have a look at writing this myself, but I don't expect to get to that in the next few months. Shall we close the issue then?
